cfme/common/provider_views.py

```diff
@@ -608,7 +608,7 @@ class ContainerProviderEditView(ProviderEditView):
     """
     @property
     def is_displayed(self):
-        return (super(ProviderEditView, self).is_displayed and
+        return (ProviderEditView.is_displayed and
```

Comment:
Required:
I think better to go with the same format for all. You can use `ContainerProviderEditView` with `super` method only.

Reply:
Thank you for the comment, I have update ContainerProviderEditView accordingly.